6.4.remember_remember.py

```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+import os.path
+
+
+def decipher(img):
```

Comment:
Dont use abbreviations in parameter names

Reply:
Done